localstack/localstack.nginx.conf

```diff
@@ -3,6 +3,6 @@ server {
   server_name localhost;
 
   location / {
-    proxy_pass http://civiform-service:9000/;
+    proxy_pass http://civiforms:9000/;
```

Comment:
Huh I didn't catch that this was civiform**s** until now.  So maybe this conf doesn't really matter much. I did test file upload in the dev deployment and it worked

Reply:
IIRC the nginx inclusion was to handle a networking bug with localstack that may be fixed now.